server/user_account/views.py

```diff
@@ -129,51 +64,50 @@ def post(self, request):
         data = request.data
         email = data.get('email', '')
         password = data.get('password', '')
+        org_id = ""
+        org_name = ""
+        response = None
 
         try:
             user = CustomUser.objects.get(email=email)
 
-            if user.is_active:
-                encrypted_password = user.password
-                is_verified = check_password(password, encrypted_password)
-                if is_verified:
-                    has_token = Token.objects.filter(user=user).count()
-                    if has_token:
-                        token = Token.objects.get(user=user)
-                    else:
-                        token = Token.objects.create(user=user)
-
-                    if user.organization is None:
-                        data = {'user': user.user_name, 'user_id': user.id, 'role': user.role,
-                                'organization': '', 'token': token.key}
-                    else:
-                        data = {'user': user.user_name, 'user_id': user.id, 'role': user.role,
-                                'organization_id': user.organization.org_id,
-                                'organization_name': user.organization.org_name,
-                                'token': token.key}
-
-                    return Response(data, status=status.HTTP_200_OK)
-
-                return Response({'detail': 'Invalid credentials'},
+            is_verified = check_password(password, user.password)
+            if is_verified and user.is_active:
+                has_token = Token.objects.filter(user=user).count()
+                if has_token:
+                    token = Token.objects.get(user=user)
+                else:
+                    token = Token.objects.create(user=user)
+                if user.organization:
+                    org_id = user.organization.org_id
+                    org_name = user.organization.org_name
+                data = {'user': user.user_name, 'user_id': user.id, 'role': user.role,
+                        'organization_id': org_id,
+                        'organization_name': org_name,
+                        'token': token.key}
+
+                response = Response(data, status=status.HTTP_200_OK)
+
+        except ObjectDoesNotExist:
+            response = None
```

Comment:
> Setting the response to None when it is already so. You can just return in the catch

I'm open to ideas here, originally I was ignoring the exception completely as this was handled by the finally block. The linters complained and thus I had to catch the exception but again it's handled by the end and the exception here is pointless but if I catch it, python complains as there are no statements after. Using a print breaks the testing line when checking it so I just reassigned. If you remove the response = None at the top, another complaint is given as response is never assigned and I wanted to avoid having multiple response statements (3 for 2 responses)